cmake/HPX_AddModule.cmake

```diff
@@ -267,10 +267,33 @@ function(add_hpx_module libname modulename)
     # cmake-format: on
   endif()
 
-  target_link_libraries(
-    hpx_${modulename} PUBLIC ${${modulename}_MODULE_DEPENDENCIES}
-  )
-  target_link_libraries(hpx_${modulename} PUBLIC ${${modulename}_DEPENDENCIES})
+  if(${modulename}_MODULE_DEPENDENCIES)
```

Comment:
Was this necessary to get things working?

Reply:
No, just for consistency purposes. I'm not sure if `target_link_libraries` can be called without arguments.

Reply:
It can indeed be empty, no need to check if it's empty.